### PR TITLE
open-image-denoise: fix build for arm64

### PR DESCRIPTION
**Pull request by Bo98**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/open-image-denoise.rb [hunk 1]
```diff
@@ -19,6 +19,9 @@ class OpenImageDenoise < Formula
   depends_on "tbb"
 
   def install
+    # Fix arm64 build targetting iOS
+    inreplace 'set(ISPC_TARGET_OS "--target-os=ios")', ""
```
Comment (carlocab) [inline Formula/open-image-denoise.rb:23]: Think you need a filename here.

Comment (Bo98) [reply Formula/open-image-denoise.rb:23]: Oops

Comment (carlocab) [reply Formula/open-image-denoise.rb]: I look forward to the day `brew` is smart enough to know which file I want to `inreplace` without my specifying

Comment (Bo98) [reply Formula/open-image-denoise.rb]: AI-powered formulae

**Review by carlocab — APPROVED** [review]

**Pull request merged**